plone-5.2.x.cfg

```diff
@@ -1,6 +1,20 @@
 [buildout]
 extends =
     base.cfg
-    http://dist.plone.org/release/5.2rc1/versions.cfg
-find-links += http://dist.plone.org/thirdparty/
+    https://dist.plone.org/release/5.2-latest/versions.cfg
```

Comment:
@ericof I personally prefer to use a specific version here. If you use "latest" the CI build can fail without notice. When you upgrade manually it is more work but at least you see on which version things break.

I guess it also would make sense to use kitconcept/buildout. I can show you that at a later point. Things are fine for now.

Reply:
I see your point about not using -latest. I personally preferred to use it *because* it would break :-) (And then I would now what needs to be done), but I can pin to 5.2.4, if you want it.
About kitconcept/buildout, perfect. Let's do it

Reply:
Done